### PR TITLE
refactor: cleanup e-show

### PR DESCRIPTION
**Pull request by codebytere**
Miscellaneous cleanup discovered when attempting to write a https://fig.io/ config for `build-tools`

### CODE REVIEW

File: src/e-show.js [hunk 1]
```diff
@@ -39,18 +39,27 @@ program.description('Show information about the current build config');
 
 program
   .command('current')
-  .description('Name of the current build config')
-  .option('-n, --no-name', "Don't show config name", false)
```
Comment (codebytere) [inline src/e-show.js:43]: This was never used.

File: src/e-show.js [hunk 2]
```diff
@@ -195,24 +198,22 @@ const gomaUrl = 'http://localhost:8088/';
 program
   .command('goma')
   .description(`Watch Goma at work at ${gomaUrl}`)
+  .option('--gn', 'Show path of the goma.gn file')
```
Comment (codebytere) [inline src/e-show.js:201]: I think it's more intuitive to combine these since they're very similar.

File: src/e-show.js [hunk 3]
```diff
@@ -154,29 +163,23 @@ program
...
-    } catch (e) {
-      fatal(e);
-    }
-  });
-
-program
-  .command('outdir')
-  .description('outdir path, e.g. "/$root/src/out/Testing"`')
+  .description('Show outdir name, e.g. "Testing"')
+  .option('--path', 'Output as JSON')
```
Comment (codebytere) [inline src/e-show.js:167]: I think it's more intuitive to make showing the `outdir` path an option under `out` 🤔

**Review by ckerr — APPROVED** [review]
These changes make sense to me.

Note, the `e show gomadir`and `e show gomagn` were originally added for https://github.com/electron/build-tools/pull/129 related to https://github.com/electron/electron/pull/22784 for CI. If this is still in use, then @jkleinsc should be aware of this PR. (If it _isn't_ still in use, maybe this section can just be removed?)

Approving because this code is fine, but also req'ing review from John

**Review by jkleinsc — CHANGES_REQUESTED** [review]
In regards to @ckerr's comment, we never made a change to call `e show gomadir` or `e show gomagn` in CI; we instead directly call:
```
node -e "console.log(require('./src/utils/goma.js').gnFilePath)"
node -e "console.log(require('./src/utils/goma.js').dir)"
```
Marking as request changes as it looks like tests aren't passing

**Comment by ckerr** [comment]
@jkleinsc requesting re-review, I've fixed up the PR.

changes that I pushed:

- removed `e show gomadir` and `e show gomagn` based on John's comment above
- fixed `e show out` 
- updated tests to use the new `--filepath` parameter
- restored `e show current`'s output format and the `--no-name` option so tests can query the config filename

**Review by jkleinsc — APPROVED** [review]

**Pull request merged**